### PR TITLE
Announcement text for archiving translations

### PR DESCRIPTION
**Pull request by purplecabbage**
Let's get this message right, and agree on it.
I have used some of the text from @erisu 's pr (#1195) but made a lot of changes. Please review/comment.

### CODE REVIEW

**Review by breautek — APPROVED** [review]

**Review by erisu — APPROVED** [review]

File: www/_posts/2021-08-25-translations.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### Declaring Translation Bankruptcy
```
Comment (erisu) [inline www/_posts/2021-08-25-translations.md:1]: I think the title should be added this way.

The build process will extract this information from this structure and fill in the title, author name, link, etc.

```suggestion
---
layout: post
author:
    name: Jesse MacFadyen
    url: https://twitter.com/purplecabbage
title:  "Declaring Translation Bankruptcy"
categories: announcements
tags: news
---

```

E.g. https://github.com/apache/cordova-docs/pull/1195/files#diff-5a1b10ca1420243c3bc1f16b19d52232f72ea7e44c803cf9b191e1bdb65f1b03R1-R8

File: www/_posts/2021-08-25-translations.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+tags: news
+---
+
+
+It has been talked about for years, and even though we [agreed to do it](https://lists.apache.org/thread.html/4768b193b66d75d1c0c7c08303c54eb0fa15894c60e99b607f7e3e5d%40%3Cdev.cordova.apache.org%3E), we never really moved on it.
+Taking things away is always harder than adding things.
+
+Back in the day, we had active contributions from the community to translate our documentation, and we had a pretty robust system to pull them all together and publish the html site.  This system has run flawlessly for years, however, the community translations slowed, then stopped, while the system still carefully churned through it all. We have been providing translated docs since Cordova CLI 3.1.0 while the maintenance of these translated docs started to diverge around Cordova CLI 6.x
+
+Over the next few weeks we will be making aggessive changes to the documentation system.
```
Comment (NiklasMerz) [inline www/_posts/2021-08-25-translations.md:17]: Maybe put this line up front to make it clear from the start what this is about.

**Review by NiklasMerz — APPROVED** [review]
Let's get this going 👍

File: www/_posts/2021-08-25-translations.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+including:
```
Comment (dblotsky) [inline www/_posts/2021-08-25-translations.md:18]: Please capitalize “include” or put in the previous sentence.

File: www/_posts/2021-08-25-translations.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+title:  "Declaring Translation Bankruptcy"
+categories: announcements
+tags: news
+---
+
+
+It has been talked about for years, and even though we [agreed to do it](https://lists.apache.org/thread.html/4768b193b66d75d1c0c7c08303c54eb0fa15894c60e99b607f7e3e5d%40%3Cdev.cordova.apache.org%3E), we never really moved on it.
+Taking things away is always harder than adding things.
+
+Back in the day, we had active contributions from the community to translate our documentation, and we had a pretty robust system to pull them all together and publish the html site.  This system has run flawlessly for years, however, the community translations slowed, then stopped, while the system still carefully churned through it all. We have been providing translated docs since Cordova CLI 3.1.0 while the maintenance of these translated docs started to diverge around Cordova CLI 6.x
```
Comment (dblotsky) [inline www/_posts/2021-08-25-translations.md:15]: Missing period at the end.

Comment (dblotsky) [reply www/_posts/2021-08-25-translations.md:15]: Also, “html” -> “HTML”.

File: www/_posts/2021-08-25-translations.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+Over the next few weeks we will be making aggessive changes to the documentation system.
+including:
+- archive all existing translated documentation to a new static `archive` site
+- remove all translations from the main repo, and update build tooling to only publish the main version
+
+We would like to thank the people who took the initiative in creating the translated docs and those who put time and effort into maintaining them.
+
+We welcome you feedback and comments.
```
Comment (dblotsky) [inline www/_posts/2021-08-25-translations.md:24]: “you feedback” -> “your feedback”.

**Review by dblotsky — COMMENTED** [review]
@purplecabbage has this already been deployed? If not, please consider fixing some typos before deploying.

Thanks to you and @erisu for writing up the blog post!

**Pull request merged**